packages/react-liveness/src/components/FaceLivenessDetector/service/utils/blazefaceFaceDetection.ts

```diff
@@ -23,13 +23,15 @@ export class BlazeFaceFaceDetection extends FaceDetection {
 
   constructor(binaryPath?: string, faceModelUrl?: string) {
     super();
-    this.faceModelUrl = faceModelUrl;
```

Comment:
this used to default to the tfhub.dev url, correct?

Reply:
correct!